doc/nrf/ug_bt_mesh_discoverable_dm_app.rst

```diff
@@ -0,0 +1,44 @@
+.. _ug_bt_mesh_discoverable_dm_app:
+
+Discovering Bluetooth mesh device in nRF Connect Device Manager
+###############################################################
+
+If your Bluetooth mesh device is not visible in the list of available devices when using the `nRF Connect Device Manager`_ mobile app (for example when performing FOTA updates), it is because the device is not advertising the SMP service UUID and the filter that only shows devices advertising this service is enabled.
+The device can still be discovered through a service discovery, for example using the `nRF Connect for Mobile`_ app.
+
+To make sure your device is visible in the `nRF Connect Device Manager`_ mobile app, do one of the following:
+
+* Disable the filter that only shows devices advertising the SMP service UUID in the `nRF Connect Device Manager`_ mobile app.
+* Make the device advertise the SMP service UUID, and thus be discoverable by the `nRF Connect Device Manager`_ mobile app with the filter enabled.
+
+Disabling the filter
+********************
+
+To disable the filter in the `nRF Connect Device Manager`_ mobile app, do the following steps:
+
+1. Tap the :guilabel:`Filter` button at the right top corner of your screen.
+#. Deselect :guilabel:`Only devices advertising SMP UUID`.
+
+You should see the device node appear in the list of devices.
+
+Advertising SMP UUID
+********************
+
+To make sure that your Bluetooth mesh device advertises the SMP service UUID, you can use the :ref:`bluetooth_mesh_light` sample where this is enabled through configuration overlay file :file:`overlay-dfu.conf`.
+Alternatively, do the following:
+
+1. Enable and configure the extended advertising API for your Bluetooth mesh sample.
+   To do so, add the following to the :file:`prj.conf` file of your sample::
+
+    CONFIG_BT_EXT_ADV=y
+    CONFIG_BT_EXT_ADV_MAX_ADV_SET=2 # 1 set for mesh, 1 set for advertising SMP server
+    CONFIG_BT_MAX_CONN=2 # 1 conn for GATT Proxy/PB-GATT, 1 conn for SMP
+
+	.. note::
```

Comment:
Is this note really necessary, I suggest to remove it. This will come in 2.2.0 where this is already true.

Reply:
Not resolved yet.